keymaps/find-and-replace.cson

```diff
@@ -20,6 +20,8 @@
 '.platform-win32 atom-text-editor, .platform-linux atom-text-editor':
   'f3': 'find-and-replace:find-next'
   'shift-f3': 'find-and-replace:find-previous'
+  'ctrl-f3': 'find-and-replace:find-next-selected'
```

Comment:
Probably good to add these to the `.platform-darwin` keymap too as <kbd>cmd-f3</kbd> and <kbd>cmd-shift-f3</kbd>.